README.md

```diff
@@ -22,14 +22,18 @@ In case you have installed Trezor Bridge previously there will be a port conflic
 #### Prerequisites
 
 You need:
-- Docker (to run NixOS in a container).
+- Docker (to run NixOS in a container). 
+- xhost
+
+On NixOS, simply enter the `nix-shell`, it is all there.
 
 #### Run it
 
-1. Run in terminal: `xhost +`
-2. Download the latest docker build: `docker-compose -f ./docker/compose.yml pull trezor-user-env-unix`
-3. Run it: `docker-compose -f ./docker/compose.yml up trezor-user-env-unix`
-4. Open http://localhost:21326.
+1. Clone this repo `git clone git@github.com:trezor/trezor-user-env.git`. If on NixOS, enter `nix-shell`.
+2. Run in terminal: `xhost +`
+3. Download the latest docker build: `docker-compose -f ./docker/compose.yml pull trezor-user-env-unix`
+4. Run it: `docker-compose -f ./docker/compose.yml up trezor-user-env-unix`
+5. Open http://localhost:21326.
```

Comment:
After merging with `incoming`, these port numbers will need to be changed.

BTW, what is the plan with the merge, is it still blocked by the CI Gitlab issues?

Reply:
Fixed by rebasing, please check.